src/debugpy/_vendored/pydevd/_pydevd_frame_eval/pydevd_frame_evaluator.template.pyx

```diff
@@ -35,10 +36,7 @@ cdef class ThreadInfo:
     # code for a function is None and there are breakpoints.
     cdef public bint force_stay_in_untraced_mode
 
-    def __init__(self):
-        cdef PyFrameObject * frame_obj
-        frame_obj = PyEval_GetFrame()
-
+    cdef initialize(self, PyFrameObject * frame_obj):
```

Comment:
This is actually a bugfix for native thread tracing (on some situations it's possible that during trace-eval `PyEval_GetFrame()` only returns null even if a frame is being executed -- one of the existing qt tests failed during this particular scenario and started failing on Github actions while it worked on Travis).